etc/setup/scripts/setup.sh

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env bash
+# -- Prerequisites --
+echo
+echo "- Generating prerequisites; -"
+
+export KC_CLIENT_ID_64=$(echo -n ${KC_CLIENT_ID} | base64)
+echo "Generated KC_CLIENT_ID_64 as ${KC_CLIENT_ID_64}"
+
+echo "- Done with prereqs.. -"
+# --
+
+echo
+echo "Setting up Keycloak;"
+source ${PWD}/etc/setup/scripts/subtasks/keycloak_setup.sh
+
+
+echo
+echo "Setting up Tyk;"
+${PWD}/etc/setup/scripts/subtasks/tyk_setup.sh
+
+echo
+echo "Setting up Vault;"
+${PWD}/etc/setup/scripts/subtasks/vault_setup.sh
+
+# TODO: OPA
+# echo
+# echo "Setting up OPA;"
+#${PWD}/etc/setup/scripts/subtasks/opa_setup.sh
+
+echo
+echo "-- AuthZ Setup Done! --"
+echo
```

Comment:
looks fine, but would recommend moving scripts specific to a service in lib/ with the service

Reply:
will look into this in a parallel dev branch